src/os/bluestore/KernelDevice.cc

```diff
@@ -46,6 +46,7 @@ KernelDevice::KernelDevice(aio_callback_t cb, void *cbpriv)
 {
   zeros = buffer::create_page_aligned(1048576);
   zeros.zero();
+  rotational = true;
```

Comment:
oh, it's fine as is then.  sorry, i'm having trouble keeping track of the changes in flight and who is working on them!